iroha/src/lib.rs

```diff
@@ -210,31 +213,33 @@ impl Iroha {
         let queue = Arc::clone(&self.queue);
         let tx_handle = task::spawn(async move {
             while let Some(transaction) = transactions_receiver.write().await.next().await {
+                log::info!("Pushing tx: {:?}", transaction);
                 if let Err(e) = queue.write().await.push_pending_transaction(transaction) {
                     log::error!("Failed to put transaction into queue of pending tx: {}", e)
                 }
             }
         });
         let queue = Arc::clone(&self.queue);
         let world_state_view = Arc::clone(&self.world_state_view);
-        let voting_handle =
-            task::spawn(async move {
-                loop {
-                    if !sumeragi.write().await.voting_in_progress().await {
-                        let is_leader = sumeragi.read().await.is_leader();
-                        sumeragi
-                            .write()
-                            .await
-                            .round(queue.write().await.get_pending_transactions(
-                                is_leader,
-                                &*world_state_view.read().await,
-                            ))
-                            .await
-                            .expect("Round failed.");
-                    }
-                    task::sleep(TX_RETRIEVAL_INTERVAL).await;
+        let voting_handle = task::spawn(async move {
+            loop {
+                if !sumeragi.write().await.voting_in_progress().await {
+                    let is_leader = sumeragi.read().await.is_leader();
+                    let transactions = queue
+                        .write()
+                        .await
+                        .get_pending_transactions(is_leader, &*world_state_view.read().await);
+
```

Comment:
Unnecessary empty lines here.

Reply:
I like to have 1 line separate from variable declaration. Here is just 1 line, so it is ok for `cargo fmt`, but I'll remove it